src/param.rs

```diff
@@ -346,6 +372,12 @@ impl Params {
         self
     }
 
+    /// Set the given paramter to the passed in `i64`.
+    pub fn set_i64(&mut self, key: Param, value: i64) -> &mut Self {
+        self.set(key, format!("{}", value));
```

Comment:
No need for `format!` here, you can use `value.to_string()` to use `Display` trait directly. The macro does the same internally.